### PR TITLE
Documentation for KIWI lock component

### PR DESCRIPTION
**Pull request by c7h**
**Description:**

Create new documentation for the KIWI lock component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

https://github.com/home-assistant/home-assistant/pull/12304

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/lock.kiwi.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description: "Instructions on how to integrate KIWI Smart Lock and Smart Entry."
+date: 2017-08-13 17:25
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Lock
+featured: false
+
+ha_release: 0.63.1
```
Comment (frenck) [inline source/_components/lock.kiwi.markdown:13]: Please update this to the next major release (`0.66` at the time of writing this comment).

File: source/_components/lock.kiwi.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+title: "KIWI Lock"
+description: "Instructions on how to integrate KIWI Smart Lock and Smart Entry."
+date: 2017-08-13 17:25
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Lock
+featured: false
+
```
Comment (frenck) [inline source/_components/lock.kiwi.markdown:12]: Maybe add the "KIWI Opening Door" logo to this PR?

**Review by frenck — CHANGES_REQUESTED** [review]
@c7h Thank you! Looks good in general, but some minor comment for you to look at. 👍

**Comment by frenck** [comment]
Rebased this PR in order to fix the build.

@c7h A little reminder, could you please take a look?

**Comment by DubhAd** [comment]
I note that the linked PR was closed 10 days ago, closing this too

**Comment by c7h** [comment]
will update and create new PR

**Pull request closed without merging**